Extreme-Programming-XP.md

```diff
@@ -1,11 +1,11 @@
 ## Extreme Programming (XP)
 
+- [Extreme Programming: A gentle introduction](http://www.extremeprogramming.org/) - by Don Wells. A good place to get started.
```

Comment:
Thank you for the PR! Could you please use [Title Case, AP style](https://capitalizemytitle.com)?

Reply:
No problem! I committed the change. Thanks for your patience since I'm new to GitHub markdown.